numpy/polynomial/tests/test_classes.py

```diff
@@ -145,14 +145,23 @@ def test_fit(Poly):
 
     def f(x):
         return x*(x - 1)*(x - 2)
+    def g(x):
+        return -(x-3)*(x-5)*(x-6)
     x = np.linspace(0, 3)
-    y = f(x)
+    y, z = f(x), g(x)
+    yz = np.column_stack((y,z))
 
     # check default value of domain and window
     p = Poly.fit(x, y, 3)
     assert_almost_equal(p.domain, [0, 3])
     assert_almost_equal(p(x), y)
     assert_equal(p.degree(), 3)
+    [q,r] = Poly.fit(x, yz, 3)
```

Comment:
```suggestion
    q, r = Poly.fit(x, yz, 3)
```
I think a tuple for the outputs is fine here.

Also, a bit of a nit on the variable naming - I'm not sure I like `q` and `r` given their "special meaning" in linear algebra (i.e. QR factorization). Maybe `p_y` and `p_z` instead? Not a blocker, just a thought.